README.md

```diff
@@ -1,2 +1,184 @@
-# Agenda NestJS
-> A NestJS module for Agenda
+# Agenda Nest
+
+<a href="https://www.npmjs.com/~agenda-nest" target="_blank"><img src="https://img.shields.io/npm/v/agenda-nest.svg" alt="NPM Version" /></a>
+<a href="https://www.npmjs.com/~agenda-nest" target="_blank"><img src="https://img.shields.io/npm/l/agenda-nest.svg" alt="Package License" /></a>
+</p>
+
+A NestJS module for Agenda
+
+## Table of Contents
+- [Background](#background)
+- [Install](#install)
+- [Configure Agenda](#configure-agenda)
+- [Job processors](#job-processors)
+- [Job schedulers](#job-schedulers)
+- [Event listeners](#event-listeners)
+- [Manually working with the queue](#manually-working-with-the-queue)
+- [License](#license)
+
+## Background
+
+Agenda Nest provides a NestJS module wrapper for [Agenda](https://github.com/agenda/agenda), a light-weight job scheduling library.  Heavily inspired by Nest's own Bull implementation, [@nestjs/bull](https://github.com/nestjs/bull), Agenda Nest provides a fully-featured implementation, complete with decorators for defining your jobs, processors and queue event listeners.  You may optionally, make use of Agenda Nest's Express controller to interface with your queues through HTTP.
+
+### Dependencies
+
+Agenda uses MongoDB to persist job data, so you'll need to have Mongo (or mongoose) installed on your system.
+
+## Install
+
+```bash
+npm install agenda-nest
+```
+
+## Configure Agenda
+
+As Agenda Nest is a wrapper for Agenda, it is configurable with same properties as the Agenda instance. Refer to [AgendaConfig](https://github.com/agenda/agenda/blob/master/lib/agenda/index.ts#L39) for the complete configuration type.
+
+```js
+import { AgendaModule } from 'agenda-nest';
+
+@Module({
+  imports: [
+    AgendaModule.register({
+      db: {
+        addresss: 'mongodb://localhost:27017',
+        collection: 'job-queue',
+      },
+    }),
+  ],
+  providers: [Jobs],
+})
+export class AppModule {}
+```
+
+## Job processors
+
+Job processors are defined using the the `@Define` decorator.  Refer to Agenda's documentation on job definition options.
```

Comment:
There are two 'the' in this sentence. Please remove one of them.

Reply:
thank you